### DISCUSSION TITLE
Tero Subtitler 1.0.1.6

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#241] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.6'>Tero Subtitler 1.0.1.6</a>.</em>

**Comment by ertugrulerzin** [comment]
Hi, 
Translator is so slowly. Can you help me?
